Batch8CucumberSeleniumAPI/src/test/resources/features/ui/Search.feature

```diff
@@ -23,4 +23,14 @@ Feature: Search functionality
 		When I search for product as "Dell"
 		And I click on any product in the Search Result
 		Then I am able to see product description and detail in new tab
-	  
\ No newline at end of file
+	  
+	  
+	  @TC0028
+	  Scenario: Verify Pagination On Product Search Page
+		Given I have browser opened and url is navigated
+		When I search for product as "Dell"
+		And I check when user is on the first page not able to click on the previous button
```

Comment:
This step needs to be as Then

Reply:
ok, Is other step is correct? or i have to add step like validate something from each page. And
If i direct search for product from suppose 4th page then pagination bar is not visible to click previous or next button.